src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -381,9 +398,16 @@ private static void ProcessResult(IAsyncResult asyncResult, AsyncCallback asyncC
 
         try
         {
-            if (activity != null && result is Exception ex)
```

Comment:
@matt-hensley The way this logic was written was causing some issues. I changed it to check first for the exception. Then in the inner logic it processes based on whether or not the activity is available.